xmodule/util/builtin_assets.py

```diff
@@ -32,47 +32,14 @@ def add_css_to_fragment(fragment, css_relative_path):
     if not css_absolute_path.is_file():
         raise FileNotFoundError(f"css file not found: {css_absolute_path}")
     css_static_path = Path('css-builtin-blocks') / css_relative_path.with_suffix('.css')
-    css_url = get_static_file_url(str(css_static_path))  # get_static_file_url is theme-aware.
```

Comment:
Note for reviewers: I removed this content because this step is _not_ theme aware. It will always return the same CSS file regardless of theme. Theme customizations will instead be injected via the CSS variables defined in `common/static/sass/_builtin-block-variables`.